tests/python_import/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -25,6 +36,22 @@ from .example import *
 
 from ..example import *
 
+import \\
+    long_module_name_1, \\
+    long_module_name_2, \\
+    long_module_name_3, \\
+    long_module_name_4
```

Comment:
Wouldn't be better to split it into multiple imports?

```py
import long_module_name_1
import long_module_name_2
import long_module_name_3
import long_module_name_4
```

Reply:
I actually had that initially, but it results in a different AST (more top-level nodes in the Module's body), and I figured that, for a tool like this, preserving the AST is the right thing to do (and probably the best safety tools we have available, to make sure we didn't mess anything up). Let me know if you disagree though!

Reply:
uhm, you're probably right!, let's leave like you did for now, we can still change it in the future :)